charts/seed-bootstrap/charts/loki/values.yaml

```diff
@@ -78,6 +83,18 @@ service:
 curator:
   port: 2718
 
+kubeProxy:
```

Comment:
Is this really kubeProxy or rather the kubeRBACProxy?

Reply:
Sorry is the kubeRBACProxy